sovrin_client/test/cli/test_tutorial_manual.py

```diff
@@ -274,11 +257,13 @@ def executeGstFlow(name, userCLI, userMap, be, connectedToTest, do, fMap,
                    syncedInviteAcceptedOutWithoutClaims, tMap,
                    transcriptClaimMap)
 
-    # TODO: restart agents to test everything still works fine
-    # faberAgent, acmeAgent, thriftAgent = restartAgents()
+    aliceCLI.looper.runFor(3)
 
-    executeGstFlow("Susan", susanCLI, susanMap, be, connectedToTest, do, fMap,
-                   aMap, jobCertificateClaimMap, newKeyringOut, reqClaimOut,
-                   reqClaimOut1, syncLinkOutWithEndpoint,
-                   syncedInviteAcceptedOutWithoutClaims, tMap,
-                   transcriptClaimMap)
+    # Same flow is executed by different cli
```

Comment:
So, right now (with latest master code), if a user (say Alice) executes the complete flow of Getting Started Tutorial, everything will work fine (as mentioned in tutorial), then say any other user (say Bob) executes the same complete flow of Getting Started tutorial, everything will still work file (even though Bob used same invitation files). If that is right expectation, then above test was to test that only.